settings.gradle.kts

```diff
@@ -121,18 +121,18 @@ include(":testing-common:integration-tests")
 include(":testing-common:library-for-integration-tests")
 
 // smoke tests
-include(":smoke-tests")
-include(":smoke-tests:images:fake-backend")
-include(":smoke-tests:images:grpc")
-include(":smoke-tests:images:play")
-include(":smoke-tests:images:quarkus")
-include(":smoke-tests:images:security-manager")
-include(":smoke-tests:images:servlet")
-hideFromDependabot(":smoke-tests:images:servlet:servlet-3.0")
-hideFromDependabot(":smoke-tests:images:servlet:servlet-5.0")
-hideFromDependabot(":smoke-tests:images:spring-boot")
+include(":smoke-tests:agent")
```

Comment:
WDYT of the following structure:
```
smoke-tests
|-- images
|-- javaagent-smoke-tests
|-- spring-starter-smoke-tests
```

Since we're changing this, we could move the javaagent tests' source code to a "leaf" module that does not include any other modules inside (like every other module in this repo).

Reply:
I would prefer having the `images` folder into an agent related folder because images are only used by the agent smoke tests.

Reply:
`fake-backend` image could be nice for non-agent tests (e.g. to test spring boot starter with the otlp exporter which is more realistic)

Reply:
So, something like this?
```
smoke-tests
|-- agent
    |-- images
|-- fake back-end
|-- spring-starter
```

The fake back end is not used today with the OTel Spring starter. So, perhaps it could stay at first in the agent folder to make it explicit it is only used with the agent smoke tests.  And it could perhaps be moved below the smoke-tests folder the day the Spring starter tests use it.

Reply:
it would be nice to also address @mateuszrzeszutek's comment:

> Since we're changing this, we could move the javaagent tests' source code to a "leaf" module that does not include any other modules inside (like every other module in this repo).